### ISSUE TITLE
Gooseneck trailer/ramp issue

### ISSUE THREAD

**Issue by MinoritySanta** [fkk-cz/noire-public#933]
This has been a problem ever since the big trailer changes, I just keep forgetting to github it

When you open the ramp on the gooseneck, the front legs collapse and it sinks the trailer (basically) into the ground.  It makes it very difficult to connect the truck if you don't have a Ram 3500.  

I have some videos at home I can post, I just wanted to get this started so I don't forget again

**Comment by Kraezia** [comment]
same problem on my end as well... the steps i follow to not encounter this error are as follows:

1. Leave gooseneck hooked up to truck
2. open the trailer
3. disconnect trailer from truck

eventually though i hope this issue is at least looked at...

**Comment by MinoritySanta** [comment]
Yeah, I actually just disconnect a little bit so it's still in the bed, put ramp down, get back in truck and re-attach so the legs drop.  Just can't get out of my truck without detaching, too much PTSD! haha

**Comment by fkk-cz** [comment]
This doesn't seem like something worth looking into, especially with the limited information you've provided and no repro steps.
